zerver/lib/test_helpers.py

```diff
@@ -419,6 +424,10 @@ def get_last_message(self):
     def get_second_to_last_message(self):
         return Message.objects.all().order_by('-id')[1]
 
+    def assertJsonableError(self, response, error_msg):
+        # type: (HttpResponse, str) -> None
+        self.assertContains(response, _(error_msg), status_code=400)
+
```

Comment:
Hmm, maybe I was confused in suggesting this -- we could just use `assert_json_error` (defined above), can't we?